### ISSUE TITLE
feat: Project creation and API keys issuing API

### ISSUE THREAD

**Issue by maxxte** [langfuse/langfuse#736]
### Describe the feature you'd like to request

While automatically setting different environments we would like to add observability in automatic way, this means to create project in langfuse and get secret and public keys, to pass them later to application

### Describe the solution you'd like to see

Introduce new [Projects](https://langfuse.com/docs/api-reference#tag/Projects) endpoint POST with parameters (project name) to create new project
And endpoint for creating API keys, which will consume project name/id and respond with secret and public keys.

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
Thanks for raising this, I understand the need. Do you suggest to introduce user-level api keys that can be used to create projects and project-level api keys?

**Comment by marcklingen** [comment]
@maxxte

**Comment by maxxte** [comment]
@marcklingen, yes, I see this as the only way to control the possibility of creating projects, thank you and sorry for longer response

**Comment by marcklingen** [comment]
Thanks for the clarification! Does a default project/user solve the problem for you or do you need to create multiple projects while and api keys while the container is already running? @maxxte
